distributed/tests/test_dask_collections.py

```diff
@@ -73,11 +73,10 @@ async def test_dataframes(c, s, a, b):
 
 
 @ignore_single_machine_warning
-@gen_cluster(client=True)
+@gen_cluster(client=True, config={"distributed.scheduler.validate": False})
```

Comment:
Why not validate here?

Reply:
It's a bug: https://github.com/dask/distributed/issues/7566